src/wide/f32x4_t.rs

```diff
@@ -48,18 +68,25 @@ impl f32x4 {
         Self::from([n, n, n, n])
     }
 
+    // TODO: What are the semantics that we want here? Fast, IEEE-754 2008 NaN
+    // Absorb, IEEE-754 2019 NaN Prop, or IEEE-754 2019 NaN Absorb?
+    // https://gist.github.com/CryZe/30cc76f4629cb0846d5a9b8d13144649
```

Comment:
max / min are interesting. If you check out my [list of platform differences](https://gist.github.com/CryZe/30cc76f4629cb0846d5a9b8d13144649) regarding max and min, you'll see that there's many different behaviors / standards. The x86 SIMD behavior differs from the WASM behavior and they also differ from f32::max. So it's a big mess. However since apparently the fact that the x86 SIMD and the f32::max fallback seem to already disagree and this apparently doesn't cause any problems, we can probably go with the fastest implementation on each platform, which are the changes I made here.